js/bybit.js

```diff
@@ -2676,6 +2812,90 @@ module.exports = class bybit extends Exchange {
         }, market);
     }
 
+    parseSpotOrder (order, market = undefined) {
```

Comment:
@ttodua I noticed this is the only parser you split, so is it safe to assume that the conflicts in the other ones are going to be minimal?

Reply:
you are right, I incorrectly misunderstood you older comment and had just done parseOrder dissection. Now I am doing all other parsers too.

Reply:
@ttodua only the ones you feel necessary pls

Reply:
> @ttodua only the ones you feel necessary pls

yes, actually I think with multi-market exchanges, the separation of parsers is very efficient - in terms of readability and possible theoretical performance (avoiding tons of extra `this.safe...` checks).